tests/e2e/source/chainsaw-test.yaml

```diff
@@ -94,12 +94,7 @@ spec:
             file: 01-workloads.yaml
         - script:
             timeout: 70s
-            content: |
-              kubectl rollout status deployment -l app=coupon
-              kubectl rollout status deployment -l app=frontend
-              kubectl rollout status deployment -l app=inventory
-              kubectl rollout status deployment -l app=pricing
-              kubectl rollout status deployment -l app=membership
+            content: ./wait_for_rollout.sh
```

Comment:
at this point, we don't yet know if the rollout has started or not, right?
can it lead to race conditions?